arm-sql/2014-04-01/swagger/sql.core.json

```diff
@@ -376,185 +376,6 @@
         }
       }
     },
-    "/subscriptions/{subscriptionId}/providers/Microsoft.Sql/servers": {
```

Comment:
This shouldn't be deleted from 2014 API. It just needs to be moved outside of sql.core.json so that compositeSql.json doesn't include the server definition from this file.